content/docs/user-guide/data-management/remote-storage/amazon-s3.md

```diff
@@ -0,0 +1,261 @@
+# Amazon S3
+
+`dvc remote add` a (new) remote name and a valid [S3] URL:
+
+```cli
+$ dvc remote add -d myremote s3://<bucket>/<key>
```

Comment:
Created [Amazon S3 page](https://dvc-org-guide-data-mgmt-leaijc.herokuapp.com/doc/user-guide/data-management/remote-storage/amazon-s3), we can use as template for the remaining types.